harmonica/visualization/prism.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2018 The Harmonica Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+"""
+Functions for visualizing prisms through pyvista
+"""
+import numpy as np
+
+try:
+    import pyvista
+except ImportError:
+    pyvista = None
+else:
+    import vtk
+
+
+def prisms_to_pyvista(prisms, properties=None):
+    """
+    Create a ``pyvista.UnstructuredGrid`` from a set of prisms
+
+    Parameters
+    ----------
+    prisms : list or 2d-array
+        List or 2d-array with the boundaries of the prisms.
+        Each row contains the boundaries of each prism in the following order:
+        ``west``, ``east``, ``south``, ``north``, ``bottom``, ``top``.
+    properties : dict or None (optional)
+        Dictionary with the physical properties of the prisms.
+        Each key should be a string and their value an array.
+        If None, no property will be added to the
+        :class:`pyvista.UnstructuredGrid`.
+        Default to None.
+
+    Returns
+    -------
+    pv_grid : :class:`pyvista.UnstructuredGrid`
+        :class:`pyvista.UnstructuredGrid` that represents the prisms with their
+        properties (if any).
+    """
+    # Check if pyvista are installed
+    if pyvista is None:
+        raise ValueError(
```

Comment:
This should be an `ImportError`

```suggestion
        raise ImportError(
```